src/event/trigger.js

```diff
@@ -10,13 +10,16 @@ define( [
 
 "use strict";
 
-var rfocusMorph = /^(?:focusinfocus|focusoutblur)$/;
+var rfocusMorph = /^(?:focusinfocus|focusoutblur)$/,
+	stopPropagationCallback = function( e ) {
```

Comment:
I wonder if we should have this inside `trigger()` since it is only used there. Opinions welcome.